packages/aws-lambda/lambdas/plain-ec2-app/setup.log

```diff
@@ -4,7 +4,7 @@ change:
   mode = APM
```

Comment:
Also out of scope, but the plain-ec2-app can be deleted as well, as it was only relevant for the demo.